tests/test_makeLimitedFitsHeader.py

```diff
@@ -29,6 +29,30 @@
 
 class MakeLimitedFitsHeaderTestCase(lsst.utils.tests.TestCase):
 
+    def assertHeadersEqual(self, header, expectedHeader, rtol=None):
+        """Compare 80 characters at a time"""
+        self.assertEqual(len(header), len(expectedHeader),
+                         msg="Compare header lengths")
+        start = 0
+        while start < len(header):
+            end = start + 80
+            # Strip trailing whitespace to make the diff clearer
+            with self.subTest("Testing individual headers"):
```

Comment:
I strongly suggest you provide the `from` and `to` strings as values for `subTest` instead of "Testing individual headers", e,g.

    this = header[start:end].rstrip()
    expected = expectedHeader[start:end].rstrip()
    with self.subTest(this=this, expected=expected):
        ...

That way you can just use assertEqual without a message (which is one big advantage to `subTest`).

I am also tempted to suggest you use `astropy.io.fits` for header comparison. The advantage is that the code would be more general (it would compare whether two headers were logically identical, without worrying about details that don't matter to FITS). That said, this code is simple enough and avoids using a 3rd party package, and since as you expect the headers to be truly identical then this is certainly acceptable and perhaps preferable.